baitap-19-10-2020/bai-6.js

```diff
@@ -0,0 +1,21 @@
+const equation = (a,b,c) => {

```

Comment:
```suggestion
const equation = (a, b, c) => {

```
tên hàm phải là động từ